### PR TITLE
added pytest-vcr in dummy test

### PR DESCRIPTION
**Pull request by chazuttu**
- added pytest-vcr in dummy test
- updated requirements-dev.txt

### CODE REVIEW

File: rece.ini [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# EJEMPLO de archivo de configuraci�n de la interfaz PyAfipWs
```
Comment (reingart) [inline rece.ini]: Please do not add this configuration file to the root directory, it could have conflicts with other devs/user .ini files.
The example should remain in the conf subdirectory
¿Do you need this file to run tests?

File: pyfepdf.py [hunk 2]
```diff
@@ -767,7 +767,7 @@ def fmt_num(self, i, fmt="%0.2f", monetary=True):
                 import locale
 
                 locale.setlocale(locale.LC_ALL, loc)
-                return locale.format(
+                return locale.format_string(
```
Comment (reingart) [inline pyfepdf.py]: In general, try to not mix unrelated changes (the PR is about the dummy method, not PDF generation)
Has this change been tested?

If not related and not tested in python 2 and python 3, it would be better if you revert the change and open another PR with it.

File: utils.py [hunk 3]
```diff
@@ -1108,7 +1108,7 @@ def norm(x, encoding="latin1"):
 def date(fmt=None, timestamp=None):
     "Manejo de fechas (simil PHP)"
     if fmt == "U":  # return timestamp
-        t = datetime.datetime.now()
+        t = datetime.datetime.utcnow()
```
Comment (reingart) [inline utils.py:1112]: Idem, if this change is not related it would be better to be in a separate PR
Also please add a comment in the code or commit description to understand why this change is needed
I remember something about the Argentina vs India timezone, for example you can add 
```suggestion
        # use universal standard time to avoid timezone differences
        t = datetime.datetime.utcnow()
```

File: pyfepdf.py [hunk 4]
```diff
@@ -1541,7 +1541,7 @@ def ProcesarPlantilla(self, num_copias=3, lineas_max=36, qty_pos="izq"):
                     else:
                         barras = ""
 
-                    f.set("CodigoBarras", unicode(barras))
+                    f.set("CodigoBarras", barras)
```
Comment (reingart) [inline pyfepdf.py]: Convert to string, just in case the user sent a integer
```suggestion
                    f.set("CodigoBarras", str(barras))
```
Anyway, as in the previous comment, maybe it would be better to open a new PR with assorted changes with small bugfixes

File: setup.py [hunk 5]
```diff
@@ -40,7 +40,7 @@
 kwargs["package_dir"] = {"pyafipws": "."}
 kwargs["packages"] = ["pyafipws"]
 opts = {}
-data_files = []
+data_files = [("pyafipws/plantillas", glob.glob("plantillas/*"))]
```
Comment (reingart) [inline setup.py]: good catch! 
This probably was lost when the setup.py was refactored to be compatible with python2/python3

**Review by reingart — APPROVED** [review]
Very good job @chazuttu !

I have left some minor comments 
To improve PR in the future it is better to submit individual short commits with related changes.
A single commit with everything is more difficult to revert or split in different PR

I have to fix test dependency installation in #54 , please pull the changes in the main branch
Sorry the test were not being automatically executed, but now you can download the report.hml from the artifacts section in checks/actions

**Comment by reingart** [comment]
- [ ] please remove the factura.pdf from the repo

Binary files could cause conflict, and there is already one in the examples directory
You could attach the generated PDF in a comment of this PR as test evidence of your changes so we can review it more easily

**Comment by reingart** [comment]
- [ ] sign off with full name (real, not nickname)

The commit was signed with "chazuttu", it would be better to use your first / last name.
You can configure your git and re-sign the commits:
```
git config user.name "FIRST_NAME LAST_NAME"
git rebase HEAD~1 --signoff
```

File: rece.ini (hunk 1 above)
Comment (chazuttu) [reply rece.ini]: No it is not needed for running tests, I will make the necessary changes.

File: pyfepdf.py (hunk 2 above)
Comment (chazuttu) [reply pyfepdf.py]: It has been tested but not related to dummy methods.

**Comment by chazuttu** [comment]
many changes in single PR
